src/hooks/useOpenFamilies.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useMemo } from 'react';
+import { useMMKVObject } from 'react-native-mmkv';
+import useAccountSettings from './useAccountSettings';
+
+export default function useOpenFamilies() {
+  const { accountAddress: address } = useAccountSettings();
+  const [openFamilies, setOpenFamilies] = useMMKVObject(
+    'open-families-' + address
```

Comment:
same nit: there seems to be no added benefit of renaming `accountAddress` to `address` and we prefer to use `accountAddress` for clarity when possible